test/helpers/assertions.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function (chai, utils) {
```

Comment:
There is a new helper being assimilated in a PR:
```zos-lib/lib/validateAddress.js```

Reply:
We can do this later once it's merged and we have that helper available